test/phabricator_test/test_phabricator.py

```diff
@@ -0,0 +1,356 @@
+from mock import patch
+import yaml
+import unittest
+from os.path import join, dirname
+try:
+    # Python 3.3 >
+    from unittest.mock import MagicMock
+except:
+    from mock import MagicMock
+from unittest import TestCase
+from reviewrot.phabricatorstack import PhabricatorService, PhabricatorReview
+from reviewrot.basereview import BaseService, BaseReview, LastComment
+import datetime
+from mock_phabricator import MockPhabricator
+
+
+PATH = 'reviewrot.phabricatorstack.'
+
+
+"""
+    This class represents the Phabricator test cases
+"""
+class PhabricatorTest(TestCase):
+
+    def setUp(self):
+        """
+            Set up the testing environment. Load mock config file
+        """
+        filename = join(dirname(__file__), "test_phabricatortest.yaml")
+        with open(filename, "r") as f:
+            self.config = yaml.safe_load(f)
```

Comment:
Just a personal preference. I would rather put loading the YAML file to the class MockPhabricator.